dependency_support/org_theopenroadproject_asap7/rc_script.tcl

```diff
@@ -12,24 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set_layer_rc -layer M1 -capacitance 6.33956E-02 -resistance 1.3889e-01
-set_layer_rc -layer M2 -capacitance 8.02750E-02 -resistance 2.4222e-02
-set_layer_rc -layer M3 -capacitance 1.23273E-01 -resistance 2.4222e-02
-set_layer_rc -layer M4 -capacitance 1.29902E-01 -resistance 1.6778e-02
-set_layer_rc -layer M5 -capacitance 1.19552E-01 -resistance 1.4677e-02
-set_layer_rc -layer M6 -capacitance 1.23114E-01 -resistance 1.0371e-02
-set_layer_rc -layer M7 -capacitance 1.05408E-01 -resistance 9.6720e-03
+# Liberty units are fF,kOhm
+set_layer_rc -layer M1 -capacitance 1.1368e-01 -resistance 1.3889e-01
```

Comment:
q: is that expected that the resistance doesn't change for the non-scaled version?

Reply:
Layers M1-9 are routing layers. Here are definitions of [scaled](https://github.com/The-OpenROAD-Project/asap7/blob/master/asap7sc7p5t_28/techlef_misc/asap7_tech_4x_201209.lef#L117-L132) and [non-scaled](https://github.com/The-OpenROAD-Project/asap7/blob/master/asap7sc7p5t_28/techlef_misc/asap7_tech_1x_201209.lef#L86-L101) M1 layers.

I assume that the layer thickness is the same in scaled and non-scaled variants and the routing wire placed on that layer is 4 times wider and longer in scaled variant than in non-scaled. In [LEF files](http://coriolis.lip6.fr/doc/lefdef/lefdefref/LEFSyntax.html#RoutingResistance), resistance is expressed in ohms per square of wire so this parameter does not depend on the changes (scaling) made to width and length of the wires on given layer.
My understanding is that if these resistances were to be specified in the LEF file, then those would have been the same for scaled and non-scaled variants.

However, in argument `-resistance`  for the command `set_layer_rc` the unit of resistance is ohm per length of default width wire. I'm not sure what is the interpretation of `default width` (perhaps the value specified in LEF tech file?) but in the docs for [set-wire-rc](https://openroad.readthedocs.io/en/latest/main/src/rsz/README.html#set-wire-rc) which handles `-resistance` similarly it is also stated that `The resistance and capacitance values are per length of wire, not per square or per square micron`.

I'm not sure if I understand that correctly but it looks reasonable to me that `ohm per length of default width wire` is not a universal unit and it should be scaled accordingly together with the width of the wires on given layer.

Reply:
The most recent and relevant change to the original file was done in https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/144. It mentions `scaling correlation` which probably refers to scaled and non-scaled variants of the PDK. The PR however changed only the capacitance while the resistance remained the same.
It is not clear to me if this is correct.
Later in https://github.com/The-OpenROAD-Project/asap7/issues/35 it was mentioned that `OpenROAD-flow-scripts` is the source of correct resistances and capacitances so maybe it is OK to use those.

@proppy please let me know what do you think about that.